pkgs/development/tools/dazel/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, buildPythonApplication, fetchPypi }:
+buildPythonApplication rec {
+  version = "0.0.36";
```

Comment:
I can see 0.0.36 is the latest version in Pypi ( https://pypi.org/project/dazel/ ), but that is ~16 months behind the releases in Github: https://github.com/nadirizr/dazel/releases .

I don't think we should package outdated software, and instead should either work with upstream to update Pypi, or we should build from the Github repo itself.